waverecovery-gui_modern.py

```diff
@@ -0,0 +1,295 @@
+import sys

```

Comment:
My IDE reports that `waverecovery-gui_modern` is an invalid file/module name (see https://peps.python.org/pep-0008/#package-and-module-names).

Probably it would be best to use an all-lowercase name without dashes and underscores, such as `waverecoverygui.py`.